common/djangoapps/student/api.py

```diff
@@ -58,8 +53,7 @@ def create_manual_enrollment_audit(
     user_email,
     transition_state,
     reason,
-    course_run_key=None,
-    role=None
+    course_run_key=None
```

Comment:
Nit: I think you can leave the trailing comma here (and in in most similar situations) to minimize the churn/delta.